python/ray/autoscaler/aws/example-full-legacy.yaml

```diff
@@ -0,0 +1,148 @@
+# An unique identifier for the head node and workers of this cluster.
+cluster_name: default
+
+# The minimum number of workers nodes to launch in addition to the head
+# node. This number should be >= 0.
+min_workers: 0
+
+# The maximum number of workers nodes to launch in addition to the head
+# node. This takes precedence over min_workers.
+max_workers: 2
```

Comment:
```suggestion
max_workers: 1
```

Reply:
err, shall I change max_workers to 1 in defaults and everywhere else as well?

Reply:
Yes  and the docs please.

Reply:
We decided to keep it to 2.